atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/DdlConfig.java

```diff
@@ -61,6 +63,16 @@ public int mutationBatchSizeBytes() {
         return 2 * 1024 * 1024;
     }
 
+    @Value.Default
+    public int parallelDdlOperationConcurrency() {
+        return 16;
```

Comment:
Concurrency here looks okay for the upgrade task described if there are only dropTableOperations. But imagine a case where you create a table and delete the same table concurrently, and delete operation is processed first instead. We end up with the table not being purged. Do we see that as a problem?